### PR TITLE
add developer info and related functions

### PR DESCRIPTION
**Pull request by UtsavK55**
Add Developer Information:
Created a form to add developer information from user and implement addDeveloper and addSkill functions

### CODE REVIEW

File: src/app.ts
Comment (02priyanshi) [inline src/app.ts:1]: remove empty file

File: src/scripts/interface/interface.ts
Comment (02priyanshi) [inline src/scripts/interface/interface.ts:1]: Not a correct way to add interface

File: src/scripts/developer.ts
Comment (02priyanshi) [inline src/scripts/developer.ts:1]: Destructure the code and implement requested changes in call

**Review by 02priyanshi — CHANGES_REQUESTED** [review]

File: src/scripts/developer.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const devAge = (document.getElementById('age') as HTMLInputElement);
+const devIsEmployed = (document.getElementById('isEmployed') as HTMLSelectElement);
+const devSkills = (document.getElementById('skills') as HTMLInputElement);
+const devExperience = (document.getElementById('experience') as HTMLInputElement);
+const devEmail = (document.getElementById('email') as HTMLInputElement);
+
+
+export const addDeveloper = (event:Event, developers:Developer[], developer:Developer)=>{
+    event.preventDefault();
+    const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```
Comment (shelly19gkmit) [inline src/scripts/developer.ts]: maintain regex in constant file

File: src/scripts/developer.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    if(!emailPattern.test(devEmail.value)){
+        alert('Please enter a valid email address.');
+        return;
+    }
+    if(isNaN(parseInt(devAge.value)) || isNaN(parseInt(devExperience.value)) || parseInt(devAge.value)< parseInt(devExperience.value)){
+        alert('Please enter correct entries');
+        return;
+    }
+    if(developers.some(item=>item.email === devEmail.value )){
```
Comment (shelly19gkmit) [inline src/scripts/developer.ts]: destructure email

File: src/scripts/developer.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    addSkill(developer.skills);
+    developer.experience = parseInt(devExperience.value);
+    developer.email = devEmail.value;
+    developers.push({...developer});
+    resetForm(developer);
+}
+export const addSkill = (skills:string[])=>{
+    const currSkill = devSkills.value.split(',');
+    currSkill.map(skill => skills.push(skill.trim()));
+}
```
Comment (shelly19gkmit) [inline src/scripts/developer.ts]: Add blank lines before/after functions

**Review by adiTrivsme — CHANGES_REQUESTED** [review]
Make sure you follow the guidelines mentioned for PR, if the commit is long (> 70) characters split it into multiple lines, details are mentioned on the PR standards section of the COC

File: src/app.ts
Comment (adiTrivsme) [inline src/app.ts:1]: remove this file

**Review by adiTrivsme — APPROVED** [review]

**Pull request merged**